### PR TITLE
Some doc improvements

### PR DESCRIPTION
**Pull request by navidcy**

### CODE REVIEW

File: examples/LatitudeLongitude_grid.jl [hunk 1]
```diff
@@ -17,36 +17,35 @@ using Oceananigans
...
 
-set!(field, (λ, φ, z) -> cosd(3λ)^2 * sind(3φ))
+set!(field, (λ, φ, z) -> sind(3λ)^2 * sind(3φ))
 
 # ### 2D visualization
 
-# We can visualize this field in 2D using a heatmap. Imaginocean.jl exports
-# a method for `heatmap!` that works with Oceananigans.jl fields.
+# We can visualize this field in 2D using a heatmap. Imaginocean.jl adds a method
+# to `heatmap!` so that it works with Oceananigans.jl fields.
```
Comment (jbisits) [inline examples/LatitudeLongitude_grid.jl:40]: fields -> `Field`s?

File: src/Imaginocean.jl [hunk 2]
```diff
@@ -182,13 +209,14 @@ end
 """
     heatsphere!(axis::Axis3, field::Field, k_index=1; kwargs...)
 
-A heatmap of an Oceananigans.jl `field` on the sphere.
+A heatmap of an Oceananigans.jl `field` on the sphere at vertical
```
Comment (jbisits) [inline src/Imaginocean.jl]: `field` -> `Field`?

File: src/Imaginocean.jl [hunk 3]
```diff
@@ -182,13 +209,14 @@ end
...
+index `k_index`.
 
 Arguments
 =========
 * `axis :: Makie.Axis3`: a 3D axis.
 * `field :: Oceananigans.Field`: an Oceananigans.jl field with non-flat horizontal dimensions.
-* `k_index :: Int`: The integer corresponding to the vertical level of the `field` to visualize; default: 1.
+* `k_index :: Int`: The integer corresponding to the vertical index of the `field` to visualize; default: 1.
```
Comment (jbisits) [inline src/Imaginocean.jl:219]: `field` -> `Field` here as well if it is changed above on line 212

**Comment by navidcy** [comment]
thanks for the suggestions!

**Pull request merged**